GA-worksheet.md

```diff
@@ -7,18 +7,18 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of the header tag is to make sure the blog has an order in it's body and we have a heading inside the page. The header tag is another code for the body, but doesn't indent.
 2. What is a Navigation Bar?
+A Navigation Bar, helps us to link another page to our blog.
+3. What tag do we use to make a navigation bar? <   a   >
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for _unwritten bullets____________________
+    <li> Stands for ___bulletpoints___________________
 
 5. What does the <a> tag do?
-
+The <a> lets us link a page to our page
 6. What would the line of HTML below do?
-
+HTML line will help our page be in a document order.
```

Comment:
This code actually gives us a hyperlink to Google.com.

The `<a>` tag sets up the link, and the `href` attribute is where the destination url goes.